lib/logfmt/encoder.ex

```diff
@@ -14,40 +51,38 @@ defmodule Logfmt.Encoder do
   @doc """
   See [`Logfmt.encode`](/logfmt/Logfmt.html#encode/1).
   """
-  @spec encode(Dict.t) :: String.t
-  def encode(list) do
-    Enum.reduce list, "", &encode_pair/2
-  end
-
-  @spec encode_pair({Dict.key, String.t | boolean | number}, String.t) :: String.t
-  defp encode_pair({key, value}, "") when is_binary(value) do
-    if String.contains?(value, "=") do
-      ~s(#{key}="#{encode_value(value)}")
-    else
-      ~s(#{key}=#{encode_value(value)})
+  @spec encode(Dict.t, options :: Keyword.t) :: String.t
+  def encode(list, options \\ []) do
```

Comment:
This option should also be exposed and documented in `Logfmt.encode/1` (now `Logfmt.encode/2`).

Reply:
Added small example of the iolist output